controllers/kubeadmbootstrapconfig_controller.go

```diff
@@ -56,3 +87,31 @@ func (r *KubeadmBootstrapConfigReconciler) SetupWithManager(mgr ctrl.Manager) er
 		For(&kubeadmv1alpha1.KubeadmBootstrapConfig{}).
 		Complete(r)
 }
+func (r *KubeadmBootstrapConfigReconciler) Map(object handler.MapObject) []reconcile.Request {
```

Comment:
Can we use`ctrl.Request` here rather than `reconcile.Request`? Would save an import methinks.

Reply:
I'll do this in a following PR, I'm not 100% convinced any of this code is right. This PR should have been just the infrastructure changes/make/yaml changes. I tossed this code in just to get go.mod right, but  probably shouldn't have.